docs/working-with-spacy.md

```diff
@@ -171,3 +171,78 @@ Then we have sentences:
 | 4 | If she wins, she will join Theresa May of Britain and Angela Merkel of Germany in the ranks of women who lead prominent Western democracies. |
 | ... | ... |
 
+## Entity Linking
+
+Unfortunately, spaCy does not provide any pre-trained Entity Linking model currently. However, we found another great
```

Comment:
entity linking should be lowercase.

We adopt the convention of each sentence on its own line - please reformat?